app/src/main/java/com/github/se/travelpouch/ui/authentication/SignIn.kt

```diff
@@ -165,6 +173,40 @@ fun SignInScreen(
                   Text("Sign in with email and password")
                 }
           }
+          if (currentUser != null) {
+            Log.d("SignInScreen", "we're here")
```

Comment:
Please update the text to a more explicit description of the state (e.g. "User already signed in") or remove this line